apps/src/sites/studio/pages/code-studio.js

```diff
@@ -92,4 +92,6 @@ setTimeout(function () {
 
 activateReferenceAreaOnLoad();
 
-window.CDOSounds = new Sounds();
+// CDOSounds is currently used in a few haml files so we need
+// to put it on window :(
+window.CDOSounds = Sounds.getSingleton();
```

Comment:
Note: Even once we don't need this on `window`, we'll want to make sure our singleton is created here or similarly early (not lazy-created) because it sets up some audio-unlock logic that's important on mobile.  See https://github.com/code-dot-org/code-dot-org/pull/5462 for context and possible testing strategies.